### PR TITLE
Added pre-commit job for cinder-operator

### PR DESCRIPTION
**Pull request by amolkahat**
This patch adds pre-commit job for cinder
operator on prow.

Depends-on: https://github.com/openstack-k8s-operators/cinder-operator/pull/147

### CODE REVIEW

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-cinder-operator-master-precommit-check

**Comment by amolkahat** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-cinder-operator-master-precommit-check

**Review by marios — COMMENTED** [review]
changes looks right but @amolkahat this needs a rebase can you please check 

also does this need to depends-on https://github.com/openstack-k8s-operators/cinder-operator/pull/147 (ie do we need to wait for that one before we merge this?)

**Comment by amolkahat** [comment]
Thanks @marios , Updated description.

**Comment by marios** [comment]
Ack thanks for update @amolkahat ... looks like the -recommit-check job failed as expected, on the issue you are fixing in the depends-on

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/35864/rehearse-35864-pull-ci-openstack-k8s-operators-cinder-operator-master-precommit-check/1638147875894464512

```
hack/configure_local_webhook.sh:14:1: E003 Indent not multiple of 4
hack/configure_local_webhook.sh:15:1: E003 Indent not multiple of 4
hack/configure_local_webhook.sh:21:1: E003 Indent not multiple of 4
hack/configure_local_webhook.sh:23:1: E003 Indent not multiple of 4 
```

which is fixed @ https://github.com/openstack-k8s-operators/cinder-operator/pull/147/files#diff-6cd5ffd3c66272b9abc4e8dc469dfb61b22aff6eaa00ac7e1f5985d8fa3116d3

File: ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml
Comment (viroel) [inline ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml:4]: needs to stay as **false**

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-cinder-operator-master-precommit-check

File: ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml
Comment (viroel) [reply ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml:4]: @bshewale can you fix this? missing this detail to merge.

**Review by viroel — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml
Comment (bshewale) [reply ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml]: Done.

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-cinder-operator-master-precommit-check

sanity check only

**Comment by viroel** [comment]
/lgtm

code looks good

**Pull request merged**
